### DISCUSSION TITLE
Documentation Contributions

### DISCUSSION THREAD

**Discussion by horacioh** [unifiedjs/unified#180] [Ideas] [answered]
Hey folks! 👋🏼

I've been building custom utilities compatible with unified. cloning repos, running tests and following the code along side the general documentation has been a good help for my journey!

there are some utils and functions that has comments that helped me, and I was wondering if contributing by adding some comments to places I believe might helped me is something y'all feel valuable? Either comments or actual documentation.

Another way to contribute was to create a tutorial for the /learn section on how to build a plugin in mutate mode for example (what I'm doing)

what y'all think?

**Comment by wooorm** [comment, marked as answer]
Generally yes, definitely appreciated!

Tthough I think *comments* should more help with maintaining code and not be a tutorial in creating similar projects, and docs should help more with answering what is this? What does it do? As you say, this might also be something for Learn. But I do think stuff in the Learn section should apply to a large share of users or be very on point (tables in markdown is one example I tihkn of).

The distinction is all a bit vague though. So I think it depend on what you’re envisioning more practically?!
